plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/CloudSdkLaunchConfigurationDelegate.java

```diff
@@ -1,213 +0,0 @@
-    final IServer server = ServerUtil.getServer(configuration);
-    if (server == null) {
-      return;
-    }
-
-    final IModule[] modules = server.getModules();
-    if (modules == null || modules.length == 0) {
-      return;
-    }
-
-    CloudSdkServerBehaviour serverBehaviour =
-        (CloudSdkServerBehaviour) server.loadAdapter(CloudSdkServerBehaviour.class, null);
-    List<String> runnables = Lists.newArrayList();
-    for (IModule module : modules) {
-      IPath deployPath = serverBehaviour.getModuleDeployDirectory(module);
-      runnables.add(deployPath.toOSString());
-    }
-
-    IRuntime runtime = server.getRuntime();
-    if (runtime == null) {
-      return;
-    }
-    IPath sdkLocation = runtime.getLocation();
-
-    serverBehaviour.setupLaunch(mode);
-
-    final CloudSdkServer cloudSdkServer = CloudSdkServer.getCloudSdkServer(server);
-    int debugPort = -1;
-    if (ILaunchManager.DEBUG_MODE.equals(mode)) {
-      final int port = getDebugPort();
-      debugPort = port;
-
-      try {
-        runDebugTarget(cloudSdkServer, modules[0].getProject().getName(), port);
-      } catch (CoreException e) {
-        Activator.logError(e);
-      }
-    }
-
-    try {
-      String commands = GCloudCommandDelegate.createAppRunCommand(sdkLocation.toOSString(),
-          runnables.toArray(new String[runnables.size()]), mode, cloudSdkServer.getApiHost(),
-          cloudSdkServer.getApiPort(), debugPort);
-
-      String additionalFlags =
-          configuration.getAttribute(CloudSdkServer.SERVER_PROGRAM_FLAGS, (String) null);
-      if (additionalFlags != null) {
-        commands += " " + additionalFlags;
-      }
-
-      // FIXME: workaround bug when running on a Java8 JVM
-      // https://github.com/GoogleCloudPlatform/gcloud-eclipse-tools/issues/181
-      commands += " --jvm_flag=-Dappengine.user.timezone=UTC";
-
-      Activator.logInfo("Executing: " + commands);
-
-      Process process = Runtime.getRuntime().exec(commands, null);
-      addProcessFactoryToLaunchConfiguration(configuration);
-      // The DebugPlugin handles the streaming of the output to the console and
-      // sends notifications of debug events
-      DebugPlugin.newProcess(launch, process, commands);
-      serverBehaviour.addProcessListener(launch.getProcesses()[0]);
-    } catch (IOException | NullPointerException | IllegalStateException | InvalidPathException e) {
-      Activator.logError(e);
-      serverBehaviour.stop(true);
-    }
-  }
-
-  /*
-   * Workaround for https://bugs.eclipse.org/bugs/show_bug.cgi?id=38016 The
-   * custom IProcessFactory creates an IProcess object that calls the /quit
-   * handler on the dev app server
-   */
-  private void addProcessFactoryToLaunchConfiguration(ILaunchConfiguration configuration) throws CoreException {
-    ILaunchConfigurationWorkingCopy workingCopy = configuration.getWorkingCopy();
-    workingCopy.setAttribute(DebugPlugin.ATTR_PROCESS_FACTORY_ID, DevAppServerProcessFactory.ID);
```

Comment:
Not yet. I'm going to have go through and cut them out. I did try removing a few more but they had connections to code that's still used, so it may take some more careful surgery to extract them.